src/components/AlertManager/AlertSteps/tests/MessageStep.test.tsx

```
@@ -266,6 +300,89 @@ describe('MessageStep', () => {
+        '.jpg,.jpeg,.png,.gif,.webp,.svg'
+      );
+    });
+
+    it('should configure max image size to 5MB', () => {
+      render(<MessageStep />);
+
+      const upload = screen.getByTestId('image-upload');
+      expect(upload).toHaveAttribute(
+        'data-max-size',
+        String(5 * 1024 * 1024)
+      );
```

ESLint

[error] Replace `⏎········'data-max-size',⏎········String(5·*·1024·*·1024)⏎······` with `'data-max-size',·String(5·*·1024·*·1024)`

(prettier/prettier)